alacritty_terminal/src/tty/unix.rs

```diff
@@ -267,9 +282,9 @@ pub fn new(config: &PtyConfig, window_size: WindowSize, window_id: u64) -> Resul
             let mut pty = Pty {
                 child,
                 file: unsafe { File::from_raw_fd(master) },
-                token: mio::Token::from(0),
+                token: 0,
                 signals,
-                signals_token: mio::Token::from(0),
+                signals_token: 0,
```

Comment:
These are just arbitrary token values unrelated to our read/write tokens I'd assume. Not great they're also zero tokens.

Reply:
Yes, these are empty tokens.